pyproject.toml

```diff
@@ -18,6 +18,19 @@ mypy = "^0.900"
 requires = ["poetry-core>=1.0.0"]
 build-backend = "poetry.core.masonry.api"
 
+[tool.pytest]
+addopts = "--cov-report html --cov-report term"
+
+[tool.coverage.run]
+branch = true
+omit = [
+    "*/setup.py",
```

Comment:
this is new, why is it?

Reply:
It appears that setup.py is also used by coverage to calculate the coverage range \shrug